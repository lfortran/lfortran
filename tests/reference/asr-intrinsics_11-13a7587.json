{
    "basename": "asr-intrinsics_11-13a7587",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_11.f90",
    "infile_hash": "22ac1dd1527a39d35236a50e3b1afe37ab40b0970f22d259a077800f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_11-13a7587.stdout",
    "stdout_hash": "51b3d2a7eadf7e0de880305f24d67bb90e94be499a5d71a03340eb2b",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}