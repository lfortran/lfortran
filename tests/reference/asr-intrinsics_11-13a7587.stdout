(TranslationUnit
    (SymbolTable
        1
        {
            intrinsics_11:
                (Program
                    (SymbolTable
                        2
                        {
                            x:
                                (Variable
                                    2
                                    x
                                    []
                                    Local
                                    (IntegerConstant 3 (Integer 4) Decimal)
                                    (IntegerConstant 3 (Integer 4) Decimal)
                                    Parameter
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            y:
                                (Variable
                                    2
                                    y
                                    [x]
                                    Local
                                    (IntrinsicElementalFunction
                                        Real
                                        [(Var 2 x)]
                                        0
                                        (Real 4)
                                        (RealConstant
                                            3.000000
                                            (Real 4)
                                        )
                                    )
                                    (RealConstant
                                        3.000000
                                        (Real 4)
                                    )
                                    Parameter
                                    (Real 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    intrinsics_11
                    []
                    [(Print
                        [(StringFormat
                            ()
                            [(Var 2 y)]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
